lib/rubocop/cop/style/redundant_condition.rb

```diff
@@ -90,9 +90,13 @@ def use_if_branch?(else_branch)
         end
 
         def else_source(else_branch)
-          wrap_else =
-            else_branch.basic_conditional? && else_branch.modifier_form?
-          wrap_else ? "(#{else_branch.source})" : else_branch.source
+          if else_branch.basic_conditional? &&
+             else_branch.modifier_form? ||
+             %i[irange erange].include?(else_branch.type)
```

Comment:
There is a `Node#range_type?` method now. 🙂 

```ruby
else_branch.range_type?
```

Reply:
Nice, that makes this much cleaner!